src/test/java/com/datasonnet/RegexTest.java

```diff
@@ -0,0 +1,79 @@
+package com.datasonnet;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RegexTest {
+
+    @Test
+    void testRegexFullMatch() throws JSONException {
+        Mapper mapper = new Mapper("DS.Regex.regexFullMatch(@'world', 'hello')", Collections.emptyList(), Collections.emptyMap(),true);
```

Comment:
I think we could move most of these tests into the inside of a DS file, with asserts in it, or alternatively parametrize the test so we don't have so much heavy duplication for running the regexes